packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx

```diff
@@ -21,6 +21,7 @@ export interface ModalDialogContentProps
   onClose: () => void
   rightPanel?: React.ReactNode
   title?: string
+  header?: React.ReactNode
```

Comment:
**question**: is this a good name for prop? or is it better to name it differently (for example, `topPanel`, `subheader`)?